tests/test_smoke.py

```diff
@@ -3096,6 +3096,7 @@ def test_multiple_buckets_creation_and_deletion(
         # Run sky storage delete all to delete all storage objects
         delete_cmd = ['sky', 'storage', 'delete']
         delete_cmd += storage_obj_name
+        delete_cmd.append('--yes')
```

Comment:
nit: for conciseness and conveying the intention more clearly.
```suggestion
        delete_cmd = ['sky', 'storage', 'delete', '--yes']
        delete_cmd += storage_obj_name
```